dictionary.txt

```diff
@@ -2,7 +2,7 @@
 \Iota	I	
 \lnot	¬	
 \neg	¬	
-^2	²	
+^2  ² 
```

Comment:
Notice that there are **tab** characters here. You removed them and put spaces.

There are supposed to only be **tab** characters.

![image](https://user-images.githubusercontent.com/20955511/135742080-eae75523-5de3-4a1f-96c7-563d11126db8.png)

```suggestion
^2	²	
```